packmanrepo.txt

```diff
@@ -0,0 +1,32 @@
+name = argparse
+	type = raw
+		url = https://raw.githubusercontent.com/SquidDev-CC/metis/dev/src/metis/argparse.lua
+		filename = argparse.lua
+	target = /usr/modules/metrics
+	category = lib
+	dependencies = none
+	version = 1.0
+	size = 5779
+end
+
+name = timerlib
+	type = raw
+		url = https://raw.githubusercontent.com/SquidDev-CC/metis/dev/src/metis/timer.lua
+		filename = timer.lua
+	target = /usr/modules/metrics
+	category = lib
+	dependencies = none
+	version = 1.0
+	size = 1466
+end
+
+name = sha1-lib
+	type = raw
+		url = https://raw.githubusercontent.com/SquidDev-CC/metis/dev/src/metis/crypto/sha1.lua
+		filename = sha1.lua
+	target = /usr/modules/metrics
```

Comment:
We currently have no cross-module dependencies, but maybe they will come in the future e.g. a module for encrypted connection will need sha-1 for storing passwords. 

Another problem with autogenerating is the version number. But this can may be resolved if we use the latest commit to the file as version number.
 
metrics is just a typo.

I will try to write a script for auto generating.